### PR TITLE
mcux-sdk: update from SDK 2.15.100 RT1180 support

### PR DESCRIPTION
**Pull request by dleach02**
Pulling in RT1180 support and various other updates to 2.15.100

### CODE REVIEW

File: mcux/mcux-sdk/boards/evkmimxrt1180/jlinkscript/evkmimxrt1180_cm33.jlinkscript
Comment (mmahadevan108) [inline mcux/mcux-sdk/boards/evkmimxrt1180/jlinkscript/evkmimxrt1180_cm33.jlinkscript:1]: This folder could be deleted

Comment (lucien-nxp) [reply mcux/mcux-sdk/boards/evkmimxrt1180/jlinkscript/evkmimxrt1180_cm33.jlinkscript:1]: Currently,  cm33/cm7 jlinkscript need to be used when debugging according.

**Review by danieldegrasse — APPROVED** [review]

**Comment by dleach02** [comment]
What did you change?

**Comment by lucien-nxp** [comment]
> What did you change?

nothing to change, only rebase to newest commit of master branch because some building error was found on other platforms on my rt1180 development branch.

**Comment by dleach02** [comment]
sdk2.16 update merged so this is not needed

**Pull request closed without merging**